memgpt/client/client.py

```diff
@@ -0,0 +1,66 @@
+from typing import Dict, List, Union
+
+from memgpt.config import AgentConfig
+from memgpt.persistence_manager import PersistenceManager
+from memgpt.server.rest_api.interface import QueuingInterface
+from memgpt.server.server import SyncServer
+
+
+class Client(object):
+    def __init__(
+        self,
+        auto_save: bool = False,
```

Comment:
I think this should probably be a property of the server too? But maybe out of scope for this PR

Reply:
I thought about that, but then it would become a state of the ws/rest servers at startup and you wouldn't be able to toggle it on/off (unless we exposed an endpoint to configure the server).  As you say, maybe something for a future PR.